velox/exec/ExchangeQueue.h

```diff
@@ -26,7 +26,8 @@ class SerializedPage {
   // Construct from IOBuf chain.
   explicit SerializedPage(
       std::unique_ptr<folly::IOBuf> iobuf,
-      std::function<void(folly::IOBuf&)> onDestructionCb = nullptr);
+      std::function<void(folly::IOBuf&)> onDestructionCb = nullptr,
+      int64_t rows = -1);
```

Comment:
@HolyLow Would it be possible to extract this change into a separate PR and use std::optional instead of magic value?

Reply:
This change is extracted to another PR: https://github.com/facebookincubator/velox/pull/8043 . Please help review it, thanks a lot.